aptos-move/framework/aptos-framework/sources/primary_fungible_store.move

```diff
@@ -178,6 +218,22 @@ module aptos_framework::primary_fungible_store {
         dispatchable_fungible_asset::transfer(sender, sender_store, recipient_store, amount);
     }
 
+    public entry fun apt_transfer(
+        sender: &signer,
+        recipient: address,
+        amount: u64,
+    ) acquires DeriveRefPod {
+        let sender_store = apt_ensure_primary_store_exists(signer::address_of(sender));
+        let recipient_store = apt_ensure_primary_store_exists(recipient);
+
+        // use internal APIs, as they skip:
+        // - owner, frozen and dispatchable checks
+        // as APT cannot be frozen or have dispatch, and PFS cannot be transfered
+        // (PFS could potentially be burned. regular transfer would permanently unburn the store.
+        // Ignoring the check here has the equivalent of unburning, transfers, and then burning again)
```

Comment:
to reviewers - want to call this out, as this biggest not-purely-optimization change, but a logical change in which invariants are checked.